internal/e2e/src/tenv.ts

```diff
@@ -713,7 +713,7 @@ export function createTenv(cwd: string): Tenv {
         abortSignal: ctrl.signal,
       });
       stream.on('data', (data) => {
-        stdboth += data.toString();
+        stdboth += data.toString().trim();
```

Comment:
Why are you trimming? The logs can intentionally contain trailing spaces as well as new lines.

Reply:
https://github.com/graphql-hive/gateway/pull/95/files#r1836745549